Sources/FTPropertyWrappers/InternetPassword.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+@propertyWrapper
+public final class InternetPassword<T: Codable>: KeychainItemPropertyWrapper<T> {
+    
+    public let serverIdentifier: String
+    
+    @QueryElement(key: kSecAttrSecurityDomain) public var domain: String?
+    @QueryElement(key: kSecAttrProtocol) public var aProtocol: String?
+    @QueryElement(key: kSecAttrAuthenticationType) public var authenticationType: String?
+    @QueryElement(key: kSecAttrPort) public var port: UInt16?
+    @QueryElement(key: kSecAttrPath) public var path: String?
+    
+    override public var itemClassIdentity: [String : Any] {
+        return [
+            kSecClass as String: kSecClassInternetPassword,
+            kSecAttrServer as String: serverIdentifier
+        ]
+    }
+
+    override public var wrappedValue: T? {
+        get { return super.wrappedValue }
```

Comment:
You can use implicit return in all the code. After all property wrappers are only available in Swift 5.1+.